.github/workflows/detect-breaking-changes.yml

```diff
@@ -17,6 +17,6 @@ jobs:
         uses: grafana/plugin-actions/is-compatible@v1
         with:
           module: './src/module.ts'
-          comment-pr: 'yes'
-          fail-if-incompatible: 'no'
```

Comment:
Am I remembering correctly that there was an issue where levity checks always fail for external contributors? Maybe a reason to keep it as just a comment?

Reply:
Yeah that might still be the case. https://raintank-corp.slack.com/archives/C01C4K8DETW/p1680805987638169
Just trying to understand here: If we have an external contribution but keep it as a comment, what is the difference for you? (Is it that it's very bad that we ever merge anything that is failing a status check?)

Reply:
I don't really love having a status failure that we ignore, it feels like better to just remove the check all together? But I guess I just don't totally follow what "fail if incompatible" will do here, since it feels like we still get a status failure even if that's set to no?

I'm game to try this change though! We can always see what happens and try again later haha

Reply:
Okay I'm hoping it is not something we will ignore most of the time at least. Hope that this change is good enough to try for now, happy to make changes later again.